recipes/scikit-umfpack/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set version = "0.2.1" %}
+
+package:
+  name: scikit-umfpack
+  version: {{ version }}
+
+source:
+  fn: scikit-umfpack--{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/s/scikit-umfpack/scikit-umfpack-{{ version }}.tar.gz
+  sha256: 9ebfb4ac4cd1ec545a9211160c65129a77b6e2fab02d8cfa30b10278f5773fd1
+  patches:
+    # osx patch below is needed in release 0.2.1, but was fixed by commit 987af60 in master
+    - setup-osx.patch  # [osx]
```

Comment:
Not sure if you know this or not, but you can tack on `.patch` or `.diff` to any commit or PR to get the corresponding patch or diff (e.g. https://github.com/scikit-umfpack/scikit-umfpack/commit/987af60.patch ).